packages/rocketchat-emojione/emojiButton.js

```diff
@@ -15,7 +17,8 @@ Template.messageBox.events({
 
 				input.val(textAreaTxt.substring(0, caretPos) + emojiValue + textAreaTxt.substring(caretPos));
 
-				input.focus();
+				if ( !window.matchMedia('all and (min-width: 360px)').matches )
+					input.focus();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'input'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2606495851/issues/source?bid=3199634&fileBranchId=3293836#l21)